packages/calypso-products/src/products-list.ts

```diff
@@ -344,6 +345,23 @@ export const JETPACK_SITE_PRODUCTS_WITH_FEATURES: Record<
 		getProductId: () => 2105,
 		getStoreSlug: () => PRODUCT_JETPACK_SEARCH_MONTHLY,
 	},
+	[ PRODUCT_JETPACK_SEARCH_FREE ]: {
+		product_name: PRODUCT_SHORT_NAMES[ PRODUCT_JETPACK_SEARCH_FREE ],
+		product_slug: PRODUCT_JETPACK_SEARCH_FREE,
+		type: PRODUCT_JETPACK_SEARCH,
+		term: TERM_ANNUALLY,
```

Comment:
Note that this is technically incorrect - the search free product has a lifetime period. However, Calypso does not have the one-time purchase period defined in a way that can be used in this product list.